kiteconnect/connect.py

```diff
@@ -634,8 +634,8 @@ def _request(self, route, method, parameters=None):
             try:
                 data = json.loads(r.content.decode("utf8"))
             except:
-                raise ex.DataException("Couldn't parse the JSON response received from the server: {contnet}".format(
-                    content=data))
+                raise ex.DataException("Couldn't parse the JSON response received from the server: {content}".format(
+                    content=r.content))
```

Comment:
Since this is a bug It would have been better if its a separate commit.

Reply:
@vividvilla Did you mean separate PR?